safe/impact_function/test/test_impact_function.py

```diff
@@ -1149,6 +1153,61 @@ def _check_minimum_fields_exists(self, impact_function):
                     inasafe_fields,
                     message.format(field_key=field['key']))
 
+    def test_equality(self):
+        """Testing IF equal operator."""
+        new_impact_function = ImpactFunction()
+        self.assertEqual(new_impact_function, new_impact_function)
+
+    @unittest.skipIf(
+        not os.path.exists('/Users/ismailsunni/dev/python/inasafe-dev'),
```

Comment:
best condition ever ;-)

Reply:
:D I am the only one who can run the test :D